### PR TITLE
Fix link to help section of cloze deletion

### LINKED ISSUE #9032: [Bug] Link to cloze deletion is wrong
Select card type as `Cloze` and click on preview. The link over there does not lead to the correct section.

### PR DESCRIPTION
**Pull request by ShridharGoel**
## Purpose / Description
Fix link to help section of cloze deletion.

## Fixes
Fixes #9032 

## Approach
Updated the link to cloze deletion.

## How Has This Been Tested?
Verified.

## Checklist
- [x] You have not changed whitespace unnecessarily (it makes diffs hard to read)
- [x] You have a descriptive commit message with a short title (first line, max 50 chars).
- [x] Your code follows the style of the project (e.g. never omit braces in `if` statements) 
- [ ] You have commented your code, particularly in hard-to-understand areas
- [x] You have performed a self-review of your own code
- [ ] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [ ] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

**Comment by david-allison-1** [comment]
Concept is good, thanks! Could you move the links to `constants.xml`

**Comment by ShridharGoel** [comment]
Updated. Removed `HELP_SITE` since it is not being used now.

File: AnkiDroid/src/main/java/com/ichi2/libanki/Collection.java [hunk 1]
```diff
@@ -1142,7 +1142,7 @@ public void updateFieldCache(String snids) {
             // empty cloze?
             if ("q".equals(type) && model.isCloze()) {
                 if (Models._availClozeOrds(model, flist, false).size() == 0) {
-                    String link = String.format("<a href=%s#cloze>%s</a>", Consts.HELP_SITE, "help");
+                    String link = String.format("<a href=%s>%s</a>", mContext.getResources().getString(R.string.link_ankiweb_docs_cloze_deletion), "help");
```
Comment (david-allison-1) [inline AnkiDroid/src/main/java/com/ichi2/libanki/Collection.java:1145]: I think the href should be quoted. Good aside from that

Comment (ShridharGoel) [reply AnkiDroid/src/main/java/com/ichi2/libanki/Collection.java]: Updated.

**Review by david-allison-1 — APPROVED** [review]
LGTM

**Review by mikehardy — APPROVED** [review]
LGTM thanks!

**Pull request merged**